websites/V/Voiranime/dist/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": {
-    "name": "DooMLorD",
-    "id": "654398152513945611"
```

Comment:
Add as contributor

Reply:
Bas allowed me to do that since the user id is no longer here (deleted or whatever?)

Reply:
Actually add as contributor the account isn't deleted.
![image](https://user-images.githubusercontent.com/34792838/126640646-bd888642-2201-4f87-8ad2-ef09e74d4798.png)

Reply:
oh my bad

Reply:
I mean give them contributor not yourself